app/queries/district_analytics_query.rb

```diff
@@ -39,7 +39,7 @@ def results
   def total_assigned_patients
     @total_assigned_patients ||=
       Patient
-        .with_hypertension
+        .for_reports
```

Comment:
👍 I guess there were quite a few places where we were missing the dead exclusions.

Reply:
Yep, found this in `CohortAnalyticsQuery` and `DistrictAnalyticsQuery`.